extensions/interactions/rule_specs_html_mapping.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I'm not really sure that this is a "rule specs html" mapping, since that implies that there's actual html somewhere. See comments above for a suggested name, but feel free to change that if you can find a name that conveys what this file actually is more clearly.

Reply:
done changed it